.github/workflows/deploy.yml

```diff
@@ -29,6 +42,17 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 11
+
+      - name: Push frontend Docker image
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: codestar-interchange-frontend
+          IMAGE_TAG: ${{ github.sha }}
```

Comment:
For the map to work, we require a variable `REACT_APP_MAPBOX_ACCESS_TOKEN` to be set. I'll Slack you the token!

Reply:
Hopefully I can configure K8S to supply that